domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/ReviewSchedule.kt

```diff
@@ -24,7 +24,8 @@ data class ReviewSchedule(
 enum class ReviewScheduleCalculationRule {
   PRISONER_READMISSION,
   PRISONER_TRANSFER,
-  LESS_THAN_6_MONTHS_TO_SERVE,
+  BETWEEN_RELEASE_AND_3_MONTHS_TO_SERVE,
+  BETWEEN_3_AND_6_MONTHS_TO_SERVE,
```

Comment:
Re: a comment in an earlier PR, I've renamed `LESS_THAN_6_MONTHS_TO_SERVE` to `BETWEEN_3_AND_6_MONTHS_TO_SERVE`; and based on the [business rules doc](https://drive.google.com/file/d/1aoGTA4npjojHgiWYxhLQewORvPrWQ2Zm/view), we have a new one `BETWEEN_RELEASE_AND_3_MONTHS_TO_SERVE`